docs/src/pages/comparing-astro-vs-other-tools.md

```diff
@@ -68,7 +68,7 @@ By contrast, Astro automatically builds your client-side JavaScript & CSS for yo
 
 Gatsby uses React to render your website. Astro is more flexible: you are free to build UI with any popular component library (React, Preact, Vue, Svelte, Solid and others) or Astro's HTML-like component syntax which is similar to HTML + JSX.
 
-Today, both Gatsby and Astro support Static Site Generation (SSG) only. Gatsby has support for incremental rebuilds, while Astro only supports full site rebuilds. Astro has stated plans to support Server-Side Rendering (SSR) in a future release, while Gatsby has no plans to support SSR.
+Today, both Gatsby and Astro support Static Site Generation (SSG) only. Gatsby has support for incremental rebuilds, while Astro only supports full site rebuilds. Astro has stated plans to support Server-Side Rendering (SSR) in a future release, while Gatsby supports SSR in version 4 up.
```

Comment:
Thanks! The rest of the paragraph no longer makes sense, can you update to match something closer to Next.js? Something like: 

> Gatsby v4 supports both Static Site Generation (SSG) and Server-Side Rendering (SSR). Today, Astro only supports Static Site Generation (SSG).

Reply:
Ok, I changed it. Should I add something on incremental rebuilds?

Reply:
I think that's a bit outside the scope of what we're talking about here so I wouldn't add it myself. But if you think its relevant, I don't have a problem with adding a few words to the end of that sentence on it:

```
Static Site Generation (SSG) and Server-Side Rendering (SSR) with incremental rebuilds.
```

Reply:
Ok, I added information on incremental rebuilds and DSR